src/maxKey.js

```diff
@@ -10,11 +10,25 @@
  */
 function maxKey(obj) {
```

Comment:
This will be a bit cleaner if you iterate over `Object.keys(obj)` and only keep track of the max key so far.